tests/fast-integration/skr-test/test.js

```diff
@@ -49,8 +50,13 @@ describe('SKR test', function() {
   });
 
 
-  // Run the OIDC tests
+  // Run the OIDC and machine type tests
   oidcE2ETest(getShootOptionsFunc, getShootInfoFunc);
+  if (process.env.MACHINE_TYPE_UPDATE) {
```

Comment:
One suggestion, instead of setting this env var for given pipelines like skr-aws true, and skr-trial false. We can always call KEB API catalog endpoint and check if the update schema has machineTypes present. 

Then PlanID is the env var that you use to check that.

Reply:
the `MACHINE_TYPE_UPDATE` is actually not a boolean but a string containing desired machine type after the update - https://github.com/kyma-project/test-infra/pull/7117, but it's not defined for every plan from the testing matrix because of the reason you mentioned above, not each plan supports update.

Reply:
But I see what you mean, I can imagine checking supported machine types from the update part of the catalog, excluding the default one and then selecting one bigger right after the default.